test/integration/32-integration-security-sync-lookup-tables.js

```diff
@@ -88,15 +88,16 @@ describe(test.testName(__filename, 3), function() {
 
     await servers[0].exchange.security.linkGroup(savedGroup, savedUser);
     await test.delay(1000);
-    let data = await testClient.data.get('/_data/historianStore/SPECIAL_DEVICE_ID_1');
+    await testClient.data.get('/_data/historianStore/SPECIAL_DEVICE_ID_1');
 
     await servers[0].exchange.security.removeLookupPath(
       'STANDARD_ABC',
       'device/OEM_ABC/COMPANY_ABC/SPECIAL_DEVICE_ID_1'
     );
+
     try {
-      data = await testClient.data.get('/_data/historianStore/SPECIAL_DEVICE_ID_1');
-      if (data) throw new Error('Test Error : Should not be authorized');
+      await testClient.data.get('/_data/historianStore/SPECIAL_DEVICE_ID_1');
+      throw new Error('Test Error : Should not be authorized');
     } catch (e) {
       test.expect(e.toString()).to.be('AccessDenied: unauthorized');
```

Comment:
tested, this catch was never happening because data was null, adjusted the test and now it looks. like the sync is failing.